### ISSUE TITLE
Adding Dan Bivins to DSVA Slack

### ISSUE THREAD

**Issue by LynneJohnson224** [department-of-veterans-affairs/va.gov-team#75645]
# Description

Please add Dan Bivins (dan.bivins@adhocteam.us) as a full member to DSVA Slack. He is joining the MHV on VA.gov team as the Accessibility Specialist.

Thank you!
